### PR TITLE
Provide a way to link to Thoth search UI with results

### PR DESCRIPTION
**Pull request by fridex**
## This introduces a breaking change

- [x] No

### CODE REVIEW

File: thoth/adviser/boots/thoth_search.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import attr
+
+from ..boot import Boot
+
+if TYPE_CHECKING:
+    from ..pipeline_builder import PipelineBuilderContext
+
+_LOGGER = logging.getLogger(__name__)
+
+_THOTH_SEARCH_URL = os.getenv("THOTH_SEARCH_ADVISER_URL", "https://thoth-station.ninja/search/{document_id}/summary")
```
Comment (Gkrumbach07) [inline thoth/adviser/boots/thoth_search.py:36]: The url was changed due to gh-pages only allowing the url to be the repo name ie `thoth-search`

```
_THOTH_SEARCH_URL = os.getenv("THOTH_SEARCH_ADVISER_URL", "https://thoth-station.ninja/search/{document_id}/summary")
```

File: tests/test_thoth_search.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    def test_verify_multiple_should_include(self, builder_context: PipelineBuilderContext) -> None:
+        """Verify multiple should_include calls do not loop endlessly."""
+        self.verify_multiple_should_include(builder_context)
+
+    def test_run(self, context: Context) -> None:
+        """Test propagating a link to Thoth Search UI."""
+        assert not context.stack_info
+
+        document_id = f"adviser-foo-bar-{random.randint(0, 1000)}"
+        search_url = "https://thoth-station.ninja/some/nice/url/to/search/{document_id}/summary/link"
```
Comment (Gkrumbach07) [inline tests/test_thoth_search.py:42]:
``` python
search_url = "https://thoth-station.ninja/some/nice/url/to/thoth-search/{document_id}/summary/link"
```

**Review by Gkrumbach07 — CHANGES_REQUESTED** [review]

File: thoth/adviser/boots/thoth_search.py (hunk 1 above)
Comment (fridex) [reply thoth/adviser/boots/thoth_search.py]: Adjusted. PTAL. Thanks! 👍🏻

Comment (fridex) [reply thoth/adviser/boots/thoth_search.py]: > LGTM

Thanks! 👍🏻

**Pull request merged**